### PR TITLE
Update to pmt noise home table

### PR DESCRIPTION
**Pull request by peter-madigan**
Reverse sort order (recent run first).
Add a column of run average QHL HHP.

### CODE REVIEW

**Comment by tlatorre-uchicago** [comment]
I get an error when running this in debug mode:

```
Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/raid/home/tlatorre/minard/minard/__init__.py", line 34, in __call__
    return self.app(environ, start_response)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/raid/home/tlatorre/minard/minard/views.py", line 810, in noise
    runs = noisedb.runs_after_run(0)
  File "/raid/home/tlatorre/minard/minard/noisedb.py", line 40, in runs_after_run
    for key in keys:
TypeError: 'NoneType' object is not iterable
```

File: minard/noisedb.py [hunk 1]
```diff
@@ -35,6 +35,7 @@ def runs_after_run(run, maxrun = '+inf'):
     Requires Redis instance, start-run and maximum run.
     '''
     keys = redis.zrangebyscore(RUN_INDEX, run, maxrun)
+    keys = keys.reverse() # Sort with most recent first
```
Comment (tlatorre-uchicago) [inline minard/noisedb.py]: reverse does an in place reverse. You can either do `keys = reversed(keys)` or `keys.reverse()`.

**Comment by peter-madigan** [comment]
Fixed.

On Sun, Apr 16, 2017 at 4:22 PM, tlatorre-uchicago <notifications@github.com

**Comment by tlatorre-uchicago** [comment]
I got another error:

```Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/raid/home/tlatorre/minard/minard/__init__.py", line 34, in __call__
    return self.app(environ, start_response)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/raid/home/tlatorre/minard/minard/views.py", line 811, in noise
    return render_template('noise.html', runs=runs)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/templating.py", line 128, in render_template
    context, ctx.app)
  File "/usr/lib/python2.6/site-packages/Flask-0.10-py2.6.egg/flask/templating.py", line 110, in _render
    rv = template.render(context)
  File "/usr/lib/python2.6/site-packages/Jinja2-2.8-py2.6.egg/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/usr/lib/python2.6/site-packages/Jinja2-2.8-py2.6.egg/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/raid/home/tlatorre/minard/minard/templates/noise.html", line 1, in top-level template code
    {% extends "layout.html" %}
  File "/raid/home/tlatorre/minard/minard/templates/layout.html", line 20, in top-level template code
    {% block body %}
  File "/raid/home/tlatorre/minard/minard/templates/noise.html", line 26, in block "body"
    <td>{{ "%.1f" % (run["average_qhl_hhp"]|float) }}</td>
  File "/usr/lib/python2.6/site-packages/Jinja2-2.8-py2.6.egg/jinja2/filters.py", line 538, in do_float
    return float(value)
UndefinedError: 'dict object' has no attribute 'average_qhl_hhp'
```

**Comment by peter-madigan** [comment]
Thanks for your patience. This one works when I test locally.

**Pull request merged**